app/css/custom.scss

```diff
@@ -871,4 +947,4 @@ blockquote {
       }
     }
   }
-}
+}
```

Comment:
`app/css/4-pages` contains css files peculiar to all pages. You could create a `_profile-page.scss` file and import it in `_-pages-index.scss` to keep things modular